src/components/search.js

```diff
@@ -99,18 +92,19 @@ class Search extends Emitter {
                      * @event Chat#$"."error"."history
                      */
                     this.chatEngine.throwError(this, 'trigger', 'search', new Error('There was a problem searching history. Make sure your request parameters are valid and history is enabled for this PubNub key.'), status);
-
                 } else {
+                    const startDate = response.startTimeToken;
+                    this.referenceDate = response.startTimeToken;
+                    this.hasMore = response.messages.length === this.config.count && startDate !== '0';
 
-                    // timetoken of the first message in response
-                    this.firstTT = response.startTimeToken;
-                    // timetoken of the last message in response
-                    this.lastTT = response.endTimeToken;
+                    response.messages.sort((left, right) => (left.timetoken < right.timetoken ? -1 : 1));
```

Comment:
Never saw variables of ```left``` and ```right``` before :P

Reply:
Well, just to keep code shorter I used common naming for comparands: left and right hand comparands.
Should this be renamed?

Reply:
@ianjennings to keep line shorter, I used common naming for comparands: left and righthand comparands.
Should I change this?

Reply:
No need to change.